src/iiiftilesource.js

```diff
@@ -239,10 +248,15 @@ $.extend( $.IIIFTileSource.prototype, $.TileSource.prototype, /** @lends OpenSea
                         $.console.error('Data has a @context property which contains no known IIIF context URI.');
                 }
             }
-            if ( !data['@id'] && data['id'] ) {
```

Comment:
I think it's fine to continue to not have this normalization as well, since the constructor is now more robust.